panel/dist/css/chat_interface.css

```diff
@@ -5,5 +5,5 @@
 }
 
 .chat-interface-input-tabs {
-  background: white;
+  background: var(--fill-color, var(--background-color, white));
```

Comment:
```suggestion
  background: var(--background-color, white);
```

Reply:
The `--background-color` is static and managed by Panel. It refers to some background?

![image](https://github.com/holoviz/panel/assets/42288570/28b69235-fe27-4edf-aa73-0df80c9cc3a7)

The `--fill-color` is dynamic and managed by the *fast design* framework. It refers to the background of the wrapping element, currently a card. But if I use `fast` elements as wrappers in custom `ReactiveHTML` components. That might be different.

![image](https://github.com/holoviz/panel/assets/42288570/947c325f-9722-4a8c-824d-9ff1820934c5)

So the `--fill-color` and `--background-color` are not the same in theory or practice.

Reply:
Thanks for the suggestion. But believe the right thing is to keep my suggestion.

Reply:
Specific design related variables should really never leak into the generic CSS definitions. Unfortunately this is currently not implemented very well, so I'll have to see if there's an alternative.

Reply:
You can use @ahuang11 suggestion right now if that is better. Then keep the issue open and we can find solution for next release.